dev/com.ibm.ws.logging.core/src/com/ibm/wsspi/collector/manager/BufferManager.java

```diff
@@ -19,12 +25,34 @@
  */
 public abstract class BufferManager {
 
-    /* Reference to ring buffer implementation */
-    protected final Buffer<Object> ringBuffer;
+	protected static final int EARLY_MESSAGE_QUEUE_SIZE=400;
```

Comment:
It might be better to move the whole EMQ logic into the BufferManagerImpl.  users of the buffer manager don't need to be aware of fields or methods related to emq (so hiding that stuff in the impl would be better).